src/patternfly/components/Switch/switch.scss

```diff
@@ -85,8 +87,8 @@
 
     &::before {
       position: absolute;
-      top: var(--pf-c-switch__toggle--Padding);
-      left: var(--pf-c-switch__toggle--Padding);
```

Comment:
`--pf-c-switch__toggle--Padding` at the top is no longer needed now that it's split.

Reply:
good catch!